composer.json

```diff
@@ -163,6 +163,7 @@
         "drupal/socialblue": "^1.0",
         "drupal/swiftmailer" : "2.0-beta1",
         "drupal/select2": "1.8",
+        "drupal/smart_trim": "1.3",
```

Comment:
@rollki I think it's better to add this depdendcy in the Dashboard module.

Reply:
Sure :)